exotic.py

```diff
@@ -407,30 +367,123 @@ def user_input(prompt, type_, val1=None, val2=None):
             return option
 
 
-def check_file_extensions(directory, fileName):
-    # Add / to end of directory if user does not input it
-    if directory[-1] != "/":
-        directory += "/"
+# Create a save directory within the current working directory
+def create_directory():
+    try:
+        directoryName = input('Enter the name for your new directory: ')
+        newDirectoryPath = os.getcwd()
+        saveDirectory = newDirectoryPath + '/' + directoryName + '/'
+        os.mkdir(saveDirectory)
+    except OSError:
+        print("Creation of the directory %s failed" % saveDirectory)
+    else:
+        print("Successfully created the directory %s " % saveDirectory)
+        return saveDirectory
```

Comment:
New directory creation function added for saving plots and anything in the future needed.